packages/core/src/app/cart/Redeemable.tsx

```diff
@@ -136,7 +136,7 @@ const RedeemableForm: FunctionComponent<
                 return <TranslatedString id="redeemable.coupon_location_error" />;
 
             default:
-                return <TranslatedString id="redeemable.code_invalid_error" />;
+                return errorMessage;
```

Comment:
errorMessage from server side is already translated

Reply:
```
errorMessage?: string
```

It seems errorMessage could be undefined, should we keep the existing message as fallback in case no error message?

Reply:
https://github.com/bigcommerce/checkout-js/pull/2075/commits/faa5895329081fbc4f979efeb425ba476f267467

yes, and also I checked if it's empty string, because if error message is empty string and we do not change to use fallback value, it will be like this:
<img width="457" alt="Screenshot 2024-10-31 at 11 44 21 AM" src="https://github.com/user-attachments/assets/f2aa3bf6-a0de-4d69-8af6-3861dea05a4f">

very confusing. 

after change it is:
<img width="457" alt="Screenshot 2024-10-31 at 11 42 09 AM" src="https://github.com/user-attachments/assets/7b765b0a-902f-426e-a9c5-d0af4e2f64bb">